lib/logstash/outputs/sumologic/sender.rb

```diff
@@ -102,8 +102,6 @@ def send_request(batch)
 
       if @stats_enabled && content.start_with?(STATS_TAG)
         body = @compressor.compress(content[STATS_TAG.length..-1])
-        headers[CATEGORY_HEADER] = "#{headers[CATEGORY_HEADER]}.stats"
```

Comment:
Rather than appending `.stats` to the category.  Can we have a separate property for this? this could be fine from a default perspective but we should always give customers full control over the source category and not force customers into hardcoded values.

Reply:
If this is already the current behavior and not a new one we introduce, I am ok with leaving as is and adding better support in future release.

Reply:
This is a quick & dirty kind of fix. We need to rethink defining of plugin metrics category, as the previous approach of appending ".stats" to payload source category will not work with templatized source category, like `%{field}`.

Reply:
Makes sense, ok we can track that in a separate issue then.

Reply:
Or instead of "rethinking" it, I could just add a new parameter `stats_category` with default value of `Logstash.stats`.

Reply:
Done